pino.js

```diff
@@ -117,7 +119,7 @@ Object.defineProperty(Pino.prototype, 'levelVal', {
     return this._levelVal
   },
   set: function setLevelVal (num) {
-    if (typeof num === 'string') { return this._setLevel(num) }
+    if (typeof num !== 'number') { return this._setLevel(num) }
```

Comment:
For one thing I need to handle the case where `num` is an object, so yes for the new feature.
For another I thought that the test was wrong anyway: `setLevelVal()` is only meant to handle numerical values so anything else should be dispatched or thrown.